entities/params/segmentsubs.go

```diff
@@ -0,0 +1,8 @@
+package params
+
+type SegmentSubs struct {
+	Ids []int64 `form:"ids[]" validate:"required"`
```

Comment:
@Dzalevski check out the `dive` parameter from validator: https://pkg.go.dev/github.com/go-playground/validator/v10#hdr-Dive

This allows us to dive into the slice and apply a validator on each element.